modules/oadp-backing-up-dpa-configuration-1-2-0.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="backing-up-dpa-configuration-1-2-0_{context}"]
+= Backing up the DPA configuration
+
+You must back up your current `DataProtectionApplication` (DPA) configuration.
+
+.Procedure
+. Save your current DPA configuration by running the following command:
++
+.Example
+[source,terminal]
+----
+$ oc get dpa -n openshift-adp -o yaml > dpa.orig.backup
+----
+. Verify that you saved the values.
```

Comment:
Unless you're going to explain how, remove this line and reformat the previous step to comply with the single-step procedure guidelines:

```
* Save your current DPA configuration by running the following command:
```